tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12131,6 +12131,26 @@ bool ValidateAttributeTargetIsFunction(Sema& S, Decl* D, const AttributeList &A)
   return false;
 }
 
+static Attr *handleHLSLGloballyCoherent(Sema &S, Decl *D,
+                                        const AttributeList &A) {
+  HLSLExternalSource *HLSLSrc = HLSLExternalSource::FromSema(&S);
+  ValueDecl *TD = dyn_cast<ValueDecl>(D);
+  if (TD && !TD->getType()->isDependentType()) {
```

Comment:
Is it possible that TD is nullptr?

Reply:
This was resolved with the most recent commit.